src/schema/v2/me/myCollection.ts

```diff
@@ -152,13 +152,21 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
   },
 }
 
-const enrichArtworks = (artworks: any, submissions: any) => {
+const enrichArtworks = (
+  artworks: any[],
+  submissions: any[],
+  filterDraftSubmissions = true
```

Comment:
Is there a need for `filterDraftSubmissions`? We always filter out draft submissions, so I don't see the need for this argument

Reply:
I wanted to make it explicit that we filter them out when we enrich the artworks. Just adding `filter(a => a.state !== 'draft')` hides the fact that we filter them out. But I'm open to changing it.

Reply:
I would add a documentation comment explaining the behaviour then. It doesn't seems right for me to add an optional attribute just to highlight the behaviour.

Reply:
👍